File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) 
    {
        // int index = nums.size() - k;
        priority_queue<int>pq(nums.begin(), nums.end());

        k = k - 1;
        while(k--)
        {
            pq.pop();
        }

        return pq.top();
    }
};

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.py
class Solution(object):
    def findKthLargest(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        l = len(nums)
        index = l - k
        heapq.heapify(nums)
        
        while index:
            heapq.heappop(nums)
            index -= 1
        
        return nums[0]
        